models/user.js

```diff
@@ -1,15 +1,15 @@
 'use strict';
 module.exports = function(sequelize, DataTypes) {
-  var User = sequelize.define('User', {
+  return sequelize.define('User', {
```

Comment:
The changes in this file surprises me - we should be using the models as defined by the create model process, unless specific changes are needed.  Do you recall what this change was for?